docs/dynamic-plugins/installing-external-plugins.md

```diff
@@ -0,0 +1,183 @@
+# Installing External Plugins
+
+RHDH supports dynamic plugins, which are plugins not included in the core RHDH distribution. These plugins can be installed or uninstalled without rebuilding the RHDH application; only a restart is required to apply the changes.
+
+If your plugin not already packaged as a dynamic plugin, you must package it into one of the supported formats before installation.
+
+## Packaging a Backstage Plugin as a Dynamic Plugin
```

Comment:
I wonder, as some of this information in [this section](https://github.com/janus-idp/backstage-showcase/blob/main/docs/dynamic-plugins/dynamic-plugins.md#preparing-dynamic-plugins-for-the-showcase) of the dynamic-plugins.md document seems relevant and similar if it should be incorporated here.

Reply:
Also it could be this part of the document could be a separate "migrating-to-dynamic-plugins.md"  And then the installing section below could be "installing-external-dynamic-plugins.md".